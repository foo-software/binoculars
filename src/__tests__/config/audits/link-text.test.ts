import auditLinkText from '../../../config/audits/link-text';

describe('link-text audit', () => {
  it('should match snapshot', () => {
    expect(auditLinkText('en')).toMatchSnapshot();
  });
});
