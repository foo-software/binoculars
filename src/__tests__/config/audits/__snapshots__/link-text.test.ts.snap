// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link-text audit should match snapshot 1`] = `
Object {
  "description": "Descriptive link text helps search engines understand your content. [Learn more](https://web.dev/link-text/).",
  "title": "Links have descriptive text",
}
`;
